### ISSUE TITLE
Can't get negative prompt / CFG to work?

### ISSUE THREAD

**Issue by CamiloMM** [oobabooga/text-generation-webui#3789]
### Describe the bug

Without messing with the regular prompt, I am trying to see if the negative prompt does anything at all. For example:
<img width="464" alt="image" src="https://github.com/oobabooga/text-generation-webui/assets/2404138/494c0319-af7d-4306-8ba9-644078b681ee">
This prompt should make the AI not express positive feelings about the color red. I tried with pronouns "I" and "You", I tried with Exllama and llama.cpp, I tried with Nous-Hermes and MythoMax, I tried with GGML and GGUF, I tried with different values for `gudance_scale`. The end result is always something like:
<img width="662" alt="image" src="https://github.com/oobabooga/text-generation-webui/assets/2404138/b6ecef73-9607-401e-9ac2-bbe0ff62a326">

In other words, the negative prompt is being completely ignored, and I don't know why, or how to fix it, or even if I'm just mistaken about what it can and can't do.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Set `guidance_scale` to 1.5, and as negative prompt use:
> _You really like red. it's your favorite color. You know what red means and care about it._

Ask the AI about the color red. It should not say it really likes it.

### Logs

```shell
No errors on console.
```

### System Info

```shell
Win10, RTX 3090, all layers on GPU for llama.cpp. Also tried with Exllama.
```

**Comment by verdelyi** [comment]
I remember a few instances where the negative prompt had an effect along the lines of its purpose, but it was never reliable for me, and it usually made the model more incoherent even when it worked. I suppose the issue is deeper and not necessarily fixable within oobabooga itself (though I'd be happy to be proven wrong).

**Comment by CamiloMM** [comment]
I'd love if someone could confirm that it doesn't actually work like Stable Diffusion, because Oobabooga and other frontends like SillyTavern present this negative prompt in a way that implies it should work. I feel like most people would put text there thinking it will work, potentially wasting tokens for what seems to be a superstition or placebo effect.
